### DISCUSSION TITLE
Nginx Proxy Manager: Testing Server Reachability - "Communication with the API failed, is NPM running correctly?"

### DISCUSSION THREAD

**Discussion by Chr1stian** [tteck/Proxmox#1992] [Help Needed]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

After installing the LXC with the script and trying to get started like I usually do with Nginx Proxy Manager, the testing for server reachability feature returns error message: 
"Communication with the API failed, is NPM running correctly?"

I have forwarded the ports get the "splash page" when I navigate to my external IP from a device outside my network.

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

<img width="492" alt="Screenshot 2023-11-01 at 19 21 01" src="https://github.com/tteck/Proxmox/assets/5319700/8109e860-aa53-4e50-8de8-6c79cb9433b7">

### Please provide detailed steps to reproduce the issue.

1. Run script to install Nginx Proxy Manager LXC
2. Try to create Lets Encypt SSL Certificate
3. Receive error as shown in image

**Comment by tteck** [comment]
My primary focus revolves around the development of application LXCs, and I'm not specialized in providing support for operational issues that may arise with these applications.

**Reply by Chr1stian** [reply]
I get and totally respect that. I appreciate finding your page and will probably use more of the LXC scripts in the coming months. However this is a basic functionality of this application, and could be related to the setup as an LXC?

**Reply by tteck** [reply]
I wouldn't expect so. Prior to switching to Cloudflared, I used NPM for years without encountering any issues. May I ask which DNS authenticator plugin you are using?

**Reply by Chr1stian** [reply]
Domeneshop. And I got the SSL certificate to work by setting the propagation seconds to 120, but shouldnt the "Test Server Reachability" button work when the ports 80/443 are open and forwarded to the LXC IP?
